packages/babel-cli/src/babel-node.js

```diff
@@ -21,7 +21,7 @@ if (argSeparator > -1) {
 }
 
 getV8Flags(function (err, v8Flags) {
-  babelArgs.forEach(function(arg){
+  babelArgs.forEach(function(arg) {
```

Comment:
Yeah its in the config https://github.com/kittens/eslint-config-kittens/blob/master/index.js#L10..